testing/bdd/features/metadata/ebuttd_required_metadata.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Converted EBUTTD file contains required metadata elements
+
+  Scenario: It parses some XML
```

Comment:
Scenario name needs updating to reflect what the scenario does, please.

Reply:
D'oh!
Thanks for spotting that one :)